src/templates/array/components/IndexCircle.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle } from '@fortawesome/pro-solid-svg-icons';
+import { Colors } from 'lattice-ui-kit';
+
+const { PURPLES } = Colors;
+
+const IconLayer = styled.span`
+  visibility: ${props => !props.visible && 'hidden'};
+  margin: 5px 0;
+`;
+
+const Index = styled.span`
+  font-size: 12px;
+  font-weight: bold;
+`;
+
+type Props = {
+  index :number;
+  visible ? :boolean;
+};
+
+const IndexCircle = ({ index, visible } :Props) => (
```

Comment:
hmm I was thinking because it's rather large and circular, rather than small and pill shaped, it would be fine to have this as a one off as it has not cropped up anywhere else in our apps. Happy to add it though.

Reply:
NVM found where it might appear in another place.

![image](https://user-images.githubusercontent.com/27182199/60929799-b8f66f00-a267-11e9-9eda-a8470a9920ad.png)

Reply:
nvm found where it might appear in another place
![image](https://user-images.githubusercontent.com/27182199/60929865-e3482c80-a267-11e9-86a9-81b23b3c1909.png)